apps/ledger-live-mobile/src/screens/SyncOnboarding/AllowManagerDrawer.tsx

```diff
@@ -25,13 +41,13 @@ const AllowManagerDrawer = ({ isOpen, device, onClose }: Props) => {
       preventBackdropClick
       noCloseButton
     >
-      <Flex mb={260} pt={110}>
-        <AllowManager
-          t={t}
-          wording={t("syncOnboarding.softwareChecksSteps.allowManagerDrawer.wording")}
-          device={device}
-          theme={theme}
-        />
+      <Flex mt={8} alignItems="center" justifyContent="center">
+        <Text fontWeight="semiBold" fontSize={24} textAlign="center">
```

Comment:
you're not supposed to specify fontSize, use a `variant` and if there is no `variant` specified in the design you should gently ask the designer to add one